### PR TITLE
Jar release

### PR DESCRIPTION
**Pull request by simao**

### CODE REVIEW

**Comment by jerrytrieu** [comment]
//cc @koshelev

File: project/SotaBuild.scala [hunk 1]
```diff
@@ -76,6 +77,25 @@ object SotaBuild extends Build {
...
 
+  lazy val nexusPassword =
+    sys.env
+      .get("NEXUS_PASSWORD")
+      .orElse(sys.props.get("nexus.password"))
+      .getOrElse("")
+
+  lazy val nexusPublish = Seq(
+    credentials += Credentials("Sonatype Nexus Repository Manager",
+      "nexus.prod01.internal.advancedtelematic.com",
```
Comment (koshelev) [inline project/SotaBuild.scala]: The changes in `rvi_sota_server` will be pushed upstream, so repository uri should be read from env, as username and password

**Comment by simao** [comment]
I can't get the tests to pass, it's failing on some db/migration error that seems unrelated. 
I made the repo credentials configurable with env vars ...

**Comment by simao** [comment]
tests are green now.

**Pull request merged**